### PR TITLE
Discord.js upgrade

### PR DESCRIPTION
**Pull request by nmoschkin**
Refactored to use the upgraded Discord.js libs.

### CODE REVIEW

File: src/utils/discord.ts [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { CommandInteraction, GuildMember, Message, EmbedBuilder, MessageReplyOpt
 import NodeCache from 'node-cache';
 
 export function getEmoteOrString(message: Message | CommandInteraction, emojiName: string, defaultString: string): string {
-	if (message instanceof CommandInteraction) {
-		if (message.guild && !message.guild.roles.everyone.permissionsIn(message.channel! as NonThreadGuildBasedChannel).has('UseExternalEmojis')) {
+	if (message instanceof CommandInteraction) {		
+		if (message.guild && (!message.guild.roles.everyone.permissionsIn(message.channel! as NonThreadGuildBasedChannel).has('UseExternalEmojis') || !message.guild?.members.me?.premiumSince)) {
```
Comment (silasary) [inline src/utils/discord.ts:7]: Bots always have external emoji permissions, but technically don't have nitro. This check will break things

**Review by silasary — CHANGES_REQUESTED** [review]

File: src/utils/discord.ts (hunk 1 above)
Comment (nmoschkin) [reply src/utils/discord.ts:7]: This was the only way that I could get emojis to show up. It would appear that the bot does not have those permissions?

Comment (nmoschkin) [reply src/utils/discord.ts:7]: I will double back and look at permissions to confirm. But I have the external emojis on, in my server...

**Review by silasary — APPROVED** [review]
Ship it 😁

**Pull request merged**